calpool-fe/src/components/SearchBar/SearchBar.css

```diff
@@ -0,0 +1,27 @@
+.input-wrapper {
+    background-color: rgb(223, 222, 222);
+    width: 100%;
+    border-radius: 30px;
+    height: 3rem;
+    padding: 0 15px;
+    box-shadow: 0px 0px 8px #ddd;
+    display: flex;
+    align-items: center;
+}
+
+input {
```

Comment:
nit: typically we don't use css on HTML elements, try to only use on class names unless all inputs are gonna look like this. it's fine in this case though since we only have one type of input